src/reader-activation-newsletters/style.scss

```
@@ -5,16 +5,11 @@
 	}
 
 	p.details {
-		margin-bottom: 0;
+		margin-bottom: 4px;
 	}
 
 	p.recipient {
-		margin-top: 4px;
-		margin-bottom: 0;
-	}
-
-	.newspack-newsletters-signup {
-		margin-top: var(--newspack-ui-spacer-5);
+		margin-top: 0;
```

```suggestion
		margin-top: 4px;
		margin-bottom: 0;
	}

	.newspack-newsletters-signup {
		margin-top: var(--newspack-ui-spacer-5);
```

So many nits! I'm sorry, Adam!